### ISSUE TITLE
[Suggestion] GTCE 1.12.2

### ISSUE THREAD

**Issue by ghost** [Quantum64/ExGregilo#52]
GregTech Community Edition aka GTCE is out so is there any plans for a 1.12.2 port

**Comment by Quantum64** [comment]
I was thinking about that but noticed that this mod already exists: https://github.com/EmosewaPixel/GregBlockUtilities

Seems to do the job of ExGregilo for CE pretty well.

**Comment by Irgendwer01** [comment]
@Quantum64 GregblockUtilities is only for Gregblock and doesn't gets updated for newer versions (ik it's an bit late lol)

**Comment by Quantum64** [comment]
Fair enough, but porting ExGregilo to CE would basically involve a complete rewrite of the mod. It just isn't something I have time for.
